src/main/java/com/gregtechceu/gtceu/api/data/chemical/material/properties/RotorProperty.java

```diff
@@ -39,7 +46,13 @@ public void setSpeed(float speed) {
         if (speed <= 0) throw new IllegalArgumentException("Rotor Speed must be greater than zero!");
         this.speed = speed;
     }
-
+    public float getEfficiency() {
+        return efficiency;
+    }
```

Comment:
This should be a `@Getter`

Reply:
Added Getters to All getBlahBlahBlahs's in RotorProperty as they had none